packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerVariables.ts

```diff
@@ -29,9 +34,11 @@ export interface DatepickerVariables {
 }
 
 export const datepickerVariables = (siteVars): DatepickerVariables => ({
+  calendarGridWidth: pxToRem(DAYS_IN_WEEK * 30),
+
   calendarCellBorder: 'none',
-  calendarCellHeight: pxToRem(32),
-  calendarCellWidth: pxToRem(32),
+  calendarCellHeight: pxToRem(30),
+  calendarCellWidth: '100%',
```

Comment:
And why calendarCell width is always 100% and headerCell width below is calculated?